kategory-core/src/main/kotlin/kategory/instances/MapKWInstances.kt

```diff
@@ -0,0 +1,36 @@
+package kategory
+
+interface MapKWMonoid<K, A> : Monoid<MapKW<K, A>> {
```

Comment:
use the new `@instance` facilities to generate the implicit objects and remove `monoid, semigroup` factories from the `MapKW.Companion`